spec/javascripts/components/story/expanded_story/expanded_story_controls_spec.js

```diff
@@ -30,7 +30,7 @@ describe('<ExpandedStoryControls />', () => {
     });
   });
 
-  describe('when the user click on delete', () => {
+  describe('when the user clicks on delete', () => {
     beforeEach(function() {
```

Comment:
You can use an arrow function here instead of a regular function
```suggestion
     beforeEach(() => {
```